pkg/apis/operator/v1alpha1/tektonhub_types.go

```diff
@@ -58,9 +58,10 @@ type DbSpec struct {
 }
 
 type ApiSpec struct {
-	HubConfigUrl  string `json:"hubConfigUrl,omitempty"`
-	ApiSecretName string `json:"secret,omitempty"`
-	RouteHostUrl  string `json:"routeHostUrl,omitempty"`
+	HubConfigUrl    string `json:"hubConfigUrl,omitempty"`
+	ApiSecretName   string `json:"secret,omitempty"`
+	RouteHostUrl    string `json:"routeHostUrl,omitempty"`
+	RefreshInterval string `json:"refreshInterval,omitempty"`
```

Comment:
can you please add a default value for this via webhook?

Reply:
I am not sure what to do here, is it just setting the default value here https://github.com/tektoncd/operator/blob/main/pkg/apis/operator/v1alpha1/tektonhub_defaults.go#L34 I have update the PR with that change then

Reply:
as this patch is adding only 1 new spec filed, i am willing to merge this as the default for that field is being added through webhook.